satgpt.py

```diff
@@ -92,14 +92,16 @@ def query_chatbot():
     try:
         # Parse the request data
         data = request.get_json()
-
+        print("here is the data in TRY ", data)
         # if query is not in data, return error
-        if "query" and "user_uuid" not in data and "r_hash" not in data:
+        if "query" not in data and "user_uuid" not in data and "r_hash" not in data:
             response = jsonify({"message": "No query provided"})
             return response, 400
 
         # check if we're in the middle of a payment
-        if "r_hash" in data:
+        elif "r_hash" in data:
+            #Is user credit checked here? 
+            print("here is the data in ELIF ", data)
```

Comment:
Let's delete all the erroneous print statements to keep our code clean. If it makes sense to keep, we'll keep it, but if it's just one-off debug logs, remove